profile.html

```diff
@@ -0,0 +1,383 @@
+                      alt=""
+                      class="profile_friends"
+                    />
+                    <p>Rita</p>
+                  </div>
+                  <div class="freinds_names">
+                    <img
+                      src="/image/pictures/pic-5.jpg"
+                      alt=""
+                      class="profile_friends"
+                    />
+                    <p>Ricky</p>
+                  </div>
+                  <div class="freinds_names">
+                    <img
+                      src="/image/pictures/pic-3.jpg"
+                      alt=""
+                      class="profile_friends"
+                    />
+                    <p>Bob_14</p>
+                  </div>
+                  <div class="freinds_names">
+                    <img
+                      src="/image/pictures/pic-6.jpg"
+                      alt=""
+                      class="profile_friends"
+                    />
+                    <p>Sally</p>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="col-12 col-lg-6 col-md-12 col-sm-12 mx-auto">
+              <div class="Stories">
+                <div>
+                  <h3><span class="details">pictures</span></h3>
+                </div>
+
+                <img
+                  src="/image/pictures/pic-5.jpg"
+                  alt=""
+                  class="profile_friends"
+                />
+
+                <img
+                  src="/image/pictures/pic-3.jpg"
+                  alt=""
+                  class="profile_friends"
+                />
+                <img
+                  src="/image/pictures/pic-4.jpg"
+                  alt=""
+                  class="profile_friends"
+                />
+
+                <img
+                  src="/image/pictures/pic-6.jpg"
+                  alt=""
+                  class="profile_friends"
+                />
+              </div>
+            </div>
+            <div class="col-12 col-lg-4 col-md-12 col-sm-12 mx-auto">
+              <div class="Stories hero_section">
+                <div>
+                  <h3><span class="details">Events</span></h3>
+                </div>
+
+                <img
+                  src="/image/event_pictures/event-group.png"
+                  alt=""
+                  class="profile_event"
+                />
+                <span>Road trip soon</span>
+                <img
+                  src="/image/event_pictures/event-header.png"
+                  alt=""
+                  class="profile_event"
+                />
+                <span>Meetup:-21-11-2022</span>
+              </div>
+            </div>
+          </div>
+        </div>
+        <!-- next section is  post feature -->
+        <div class="container">
+          <div class="row">
+            <div class="center">
+              <div class="col-12 col-sm-12 col-md-8">
+                <form class="post">
```

Comment:
Form validation + form action would be a good improvement here

Reply:
i will  change now